db/utility.js

```diff
@@ -96,7 +96,7 @@ const createOrder = async (order)=>{
         await dbMutation(`INSERT INTO \`Order\` VALUES('${order.id}', '${order.tableNo}', ${order.totalPrice}, '${order.time}' )`)
         order.items.forEach(async (item)=>{
             let Order_Item_InfoId = order.id+"_"+item.id
-            await dbMutation(`INSERT INTO \`Order_Item_Info\` VALUES('${Order_Item_InfoId}', '${order.id}', '${item.id}', '${item.quantity}', '${item.note}')`)
+            await dbMutation(`INSERT INTO \`Order_Item_Info\` VALUES('${Order_Item_InfoId}', '${order.id}', '${item.id}', '${item.quantity}', '${item.note}'), 'unready')`)
```

Comment:
lol i was going to suggest to use enum for the status, but i see there are no enums in JavaScript 😭. Seems like we should've used TypeScript )))

maybe at least put these statuses into constants in a separate file? like so
```
const STATUS_UNREADY = "unready"
``` 
and then pass this constant into the method